en/docs/learn/rate-limiting/access-control.md

```diff
@@ -18,35 +18,36 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 2.  Under the **Rate Limiting Policies** tab and navigate to **Advanced Policies.**
 3.  Click Add New Policy to add a new Rate Limit tier.
 
-    ![](../../assets/img/learn/allowed-ip-add-policy.png)
+    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
 
-4.  Fill the details as below and click **Add Conditional Group** .
+4.  Fill the details as below and click **Add Conditional Group**.
 
-    ![](../../assets/img/learn/allowed-ip-add-conditional-group.png)
+    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)
 
 5.  Open the Conditional Group added and fill the details.
 
     | Property            | Value                                                                                                          |
     |---------------------|----------------------------------------------------------------------------------------------------------------|
-    | IP Condition Policy | Checked                                                                                                        |
     | IP Condition Type   | Specific IP                                                                                                    |
     | IP Address          | <IP_Address_to_be_allowed> E.g. 193.100.3.106                                                              |
     | Invert Condition    | Checked (If Invert Condition check then condition only apply to the IPs which not mention in IP Address above) |
```

Comment:
Please change description to something similar to - 
If the 'Invert Condition' is checked, then the condition applies only to the IPs not mentioned in the 'IP Address'.

Reply:
Fixed with 8ac6250